playbooks/common/openshift-cluster/upgrades/v4_0/upgrade_control_plane_part2.yml

```diff
@@ -116,16 +116,3 @@
   - import_role:
       name: openshift_web_console
       tasks_from: remove_old_asset_config.yml
-
-# This is a one time migration. No need to save it in the 3.11.
```

Comment:
Should this be removed in 3.11?

Reply:
Yeah, I'll include this in https://github.com/openshift/openshift-ansible/pull/10031 and test it there

Reply:
Given that a lot of this should go away in 4.0 I'm not terribly worried about doing a lot of immediate cleanup. Just that we can get 3.11 to 4.0 tests into CI until such a time that we begin active 4.0 development in openshift-ansible.